docs/source/Support/bskReleaseNotes.rst

```diff
@@ -734,7 +734,7 @@ Version 2.1.0 (Nov. 13, 2021)
 - Fixed a rare issue where RW data didn't stick
 - Fixed an issue subscribing to a C++ wrapped message object from python
 - Cleaned up documentation on using datashaders and bokeh to interactively plot large simulation data sets.
-  The script :ref:`scenarioAnalyzeMonteCarlo` is updated to discuss the particular challenges in running this
+  The script :ref:`scenarioVisualizeMonteCarlo` is updated to discuss the particular challenges in running this
```

Comment:
You are changing the release notes entry from BSK v2.0.1.  But, the statement doesn't make sense now as the current folder is fully changed.  To avoid the RST build documentation link issue to a defunct folder, just replace
```
:ref:`scenarioAnalyzeMonteCarlo`
``` 
with 
```
``scenarioAnalyzeMonteCarlo``
```